src/commands/show/user.ts

```diff
@@ -27,11 +30,22 @@ export default class ShowUser extends Command {
 
   public async run(): Promise<void> {
     const { flags } = await this.parse(ShowUser)
-    const { consoleAuthToken: token } = getSession()
-    if (!token) {
+    if (!isAuthenticated()) {
       throw new CliError(Unauthorized, StatusCodes.UNAUTHORIZED, 'userManagement')
     }
+    const { consoleAuthToken: token, account } = getSession()
+    const analyticsData: EventDTO = {
+      name: 'CLI_USER_INFO_SHOWED',
```

Comment:
since category and component are always the same it makes sense to add a wrapper method for analyticsService (in a follow-up PR of course)

Reply:
Yes